hwdb.d/60-sensor.hwdb

```diff
@@ -529,6 +529,10 @@ sensor:modalias:acpi:BOSC0200*:dmi:*:svnLINX*:pnLINX1010B:*
 sensor:modalias:acpi:KIOX000A*:dmi:*:svnLINX*:pnLINX12*64:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
 
+# Linx Vision 8
+sensor:modalias:acpi:KIOX000A*:dmi:*:svnLINX:pnVISION004:*
```

Comment:
please merge this with the previous entry, i.e. place two `sensor:…` lines one after the other, and then one body for it... See the IdeaPad lines further up for an example

These devices identify in a very similar way, and have the same matrix, hence merging the entries makes a lot of sense

Reply:
Done.